main.py

```diff
@@ -5,8 +5,9 @@
 app.config["DEBUG"] = True
 
 def check_character_len(string):
-    if int(len(string)) > 20 or int(len(string)) < 3:
+    if len(string) > 20 or len(string) < 3:
```

Comment:
No need to type cast the output from the `len()` function, it can only return integers.